tensorflow_model_optimization/python/core/common/keras/compression/algorithms/BUILD

```diff
@@ -43,3 +46,27 @@ py_test(
         # tensorflow dep1,
     ],
 )
+
+py_library(
+    name = "weight_clustering",
+    srcs = ["weight_clustering.py"],
+    srcs_version = "PY3",
+    deps = [
+        # tensorflow dep1,
+        "//tensorflow_model_optimization/python/core/clustering/keras:clustering_centroids",
+        "//tensorflow_model_optimization/python/core/clustering/keras:clustering_registry",
+        "//tensorflow_model_optimization/python/core/common/keras/compression:algorithm",
+    ],
+)
+
+py_test(
+    name = "weight_clustering_test",
+    timeout = "eternal",
```

Comment:
We should remove this - if the test takes this long, there's a problem.

Reply:
Done.